NGit/NGit.Api.Errors/PatchApplyException.cs

```diff
@@ -65,5 +65,9 @@ public class PatchApplyException : GitAPIException
 		public PatchApplyException(string message) : base(message)
 		{
 		}
-	}
+
+		public PatchApplyException()
+        {
+        }
```

Comment:
Hmm, looks fine in Visual Studio - maybe a mixture of tabs and spaces. I'll try to fix it.